Tests/RegressionTests.cs

```diff
@@ -288,7 +258,50 @@ public void ParameterizedAlgorithm()
                 {"Tracking Error", "0.171"},
                 {"Treynor Ratio", "4.634"},
                 {"Total Fees", "$3.09"},
-            });
+            };
+
+            return new List<AlgorithmStatisticsTestParameters>
+            {
+                // CSharp
+                new AlgorithmStatisticsTestParameters("AddRemoveSecurityRegressionAlgorithm", addRemoveSecurityRegressionStatistics, Language.CSharp),
+                new AlgorithmStatisticsTestParameters("BasicTemplateAlgorithm", basicTemplateStatistics, Language.CSharp),
+                new AlgorithmStatisticsTestParameters("CustomDataRegressionAlgorithm", customDataRegressionStatistics, Language.CSharp),
+                new AlgorithmStatisticsTestParameters("DropboxBaseDataUniverseSelectionAlgorithm", dropboxBaseDataUniverseSelectionStatistics, Language.CSharp),
+                new AlgorithmStatisticsTestParameters("DropboxUniverseSelectionAlgorithm", dropboxUniverseSelectionStatistics, Language.CSharp),
+                new AlgorithmStatisticsTestParameters("LimitFillRegressionAlgorithm", limitFillRegressionStatistics, Language.CSharp),
+                new AlgorithmStatisticsTestParameters("ParameterizedAlgorithm", parameterizedStatistics, Language.CSharp),
+                new AlgorithmStatisticsTestParameters("RegressionAlgorithm", regressionStatistics, Language.CSharp),
+                new AlgorithmStatisticsTestParameters("UniverseSelectionAlgorithm", universeSelectionRegressionStatistics, Language.CSharp),
```

Comment:
The `UniverseSelectionAlgorithm` throws null reference because no CSharp type exists, I think it should be `UniverseSelectionRegressionAlgorithm`